Create Getting started with List draft (#1925)
DevExpress/devextreme-documentation: concepts/05 UI Components/List/00 Getting Started with List/06 Search Items.md

> To add a search bar to the **List**, set the [searchEnabled](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchEnabled) property to true. Use the [searchMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchMode) property to specify one of the following modes: `contains`, `startswith`, and `equals`.

Reviewer: When you are talking about search modes, it's unclear what they do.

```suggestion
To add a search bar to the **List**, set the [searchEnabled](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchEnabled) property to **true**. Use the [searchMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchMode) property to specify whether found list items should start with, contain, or match the search string.
```

> To add a search bar to the **List**, set the [searchEnabled](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchEnabled) property to true. Define the [searchMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchMode) property and specify a search mode. The search bar also required a data field name or several names to search by. Define the [searchExpr](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchExpr) property and assign a field name or an array of names to it.

Reviewer: > Define the [searchMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchMode) property and specify a search mode.

Maslo maslyanoe :) Let's describe available search modes.

>  The search bar also required a data field name or several names to search by. Define the [searchExpr](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchExpr) property and assign a field name or an array of names to it.

This functionality is used rarely. We can skip it.

---
##### jQuery

    <!-- tab: index.js -->
    $(function () {
        $('#list1').dxList({
            // ...
            searchEnabled: true,
            searchMode: 'contains',
            searchExpr: ['Name'],
        });

>             // ...

Reviewer: ```suggestion
```

To add a search bar to the **List**, set the [searchEnabled](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchEnabled) property to **true**. Use the [searchMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#searchMode) property to specify whether found list items should start with, contain, or match the search string.

---
##### jQuery

    <!-- tab: index.js -->
    $(function () {
        $('#list').dxList({
            // ...
            searchEnabled: true,
            searchMode: 'contains',
        });
    });
##### Angular

    <!-- tab: app.component.html -->
    <dx-list...
        [searchEnabled]="true"
        searchMode="contains">
    </dx-list>

##### Vue

    <!-- tab: App.vue -->
    <template>
        <div id="app-container">
            <DxList...
                :search-enabled="true"
                search-mode="contains">
            </DxList>
        </div>
    </template>

    <script>
    // ...
    import service from './products.service';

    export default {
        // ...
        data() {
            // ...
        },

>     }

Reviewer: ```suggestion
    // ...
```
